src/kili/utils/labels/geojson/point.py

```diff
@@ -0,0 +1,123 @@
+"""Point label utils."""
+from typing import Any, Dict, List, Literal, Optional, Union
+
+
+def kili_point_to_geojson_point(
+    point: Dict[str, float]
+) -> Dict[Literal["type", "coordinates"], Union[Literal["Point"], List[float]]]:
+    """Convert a Kili point to a geojson point.
+
+    Args:
+        point: a Kili point (vertex).
+
+    Returns:
+        A geojson point.
+
+    !!! Example
+        ```python
+        >>> point = {"x": 1.0, "y": 2.0}
+        >>> kili_point_to_geojson_point(point)
+        {
+            "type": "Point",
+            "coordinates": [1.0, 2.0]
+        }
+        ```
+    """
+    return {"type": "Point", "coordinates": [point["x"], point["y"]]}
+
+
+def kili_point_annotation_to_geojson_point_feature(
+    point_annotation: Dict[str, Any], job_name: Optional[str] = None
+) -> Dict[str, Any]:
+    """Convert a Kili point annotation to a geojson point feature.
+
+    Args:
+        point_annotation: a Kili point annotation.
+        job_name: the name of the job to which the annotation belongs.
+
+    Returns:
+        A geojson point feature.
+
+    !!! Example
+        ```python
+        >>> point = {
+            'children': {},
+            'point': {'x': -79.0, 'y': -3.0},
+            'categories': [{'name': 'A'}],
+            'mid': 'mid_object',
+            'type': 'marker'
+        }
+        >>> kili_point_annotation_to_geojson_point_feature(point)
+        {
+            'type': 'Feature',
+            'geometry': {
+                'type': 'Point',
+                'coordinates': [-79.0, -3.0]},
+                'id': 'mid_object',
+                'properties': {
+                    'kili': {
+                        'categories': [{'name': 'A'}],
+                        'children': {},
```

Comment:
what is the point of writing children too ?

Reply:
I just copy what's in the kili annotation